Description: Eliminated the exit with the insert key.
  The change in line 514 is from the fix-config-keys.patch patch
Author: Innocent De Marchi <tangram.peces@gmail.com>
Bug-Debian: http://bugs.debian.org/555251
Last-Update: 2018-03-16

--- pinball-0.3.1.orig/src/Pinball.cpp	2003-11-20 17:46:17.000000000 +0100
+++ pinball-0.3.1/scr/Pinball.cpp	2018-03-16 19:22:50.844304579 +0100
@@ -247,6 +247,7 @@
     case 4: w = 800; h = 600; break;
     case 5: w = 1024; h = 768; break;
     case 6: w = 1280; h = 1024; break;
+    case 7: w = 1920; h = 1200; break;
     default: w = 640; h = 480;
     }
     if (config->getWidth() != w) {
@@ -441,6 +442,8 @@
     menusize->setCurrent(5);
   } else 	if (Config::getInstance()->getWidth() == 1280) {
     menusize->setCurrent(6);
+  } else 	if (Config::getInstance()->getWidth() == 1920) {
+    menusize->setCurrent(7);
   } else {
     menusize->setCurrent(3);
   }
@@ -511,7 +514,8 @@
   menucfg->addMenuItem(menuaudio);
 
   MenuSub* menukey = new MenuSub("keyboard", engine);
-  menukey->setBottomText("shorcuts for view change F5-F8");
+  //menukey->setBottomText("shorcuts for view change F5-F8");
+  menukey->setBottomText("L-eft R-ight B-ottom F-lip N-udge");
   menucfg->addMenuItem(menukey);
 
   string filename = string(Config::getInstance()->getDataSubDir()) + "/splash.png";
@@ -615,6 +619,7 @@
   menusize->addText(  "screen size:  800x600");
   menusize->addText(  "screen size: 1024x768");
   menusize->addText(  "screen size:1280x1024");
+  menusize->addText(  "screen size:1920x1200");
   menugfx->addMenuItem(menusize);
 
   menufilter = new MenuChoose(engine);
